src/main/java/seedu/duke/assets/Doctor.java

```diff
@@ -1,17 +1,23 @@
 package seedu.duke.assets;
 
 public class Doctor extends Person {
-    String specialization;
+    private String nric;
+    private String fullName;
+    private int age;
+    private char gender;
+    private String address;
+    private String dob;
```

Comment:
There's no need for these variables to be in the Doctor class as it is already in the superclass Person

Reply:
yup oki tks